doc/tutorials/transformer/model_retrieval.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<p>a short sentence here would be nice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>